client/package.json

```diff
@@ -42,6 +42,7 @@
 		"@automattic/global-styles": "workspace:^",
 		"@automattic/help-center": "workspace:^",
 		"@automattic/i18n-utils": "workspace:^",
+		"@automattic/jetpack-ai-calypso": "workspace:^",
```

Comment:
Let's add it as a dependency only if we actually need to use it in Calypso's `client`.

Reply:
It will be needed soon, but not in this PR. Removed.